app/src/main/java/com/rtdti/calc16/CalcMath.kt

```diff
@@ -186,8 +187,9 @@ initial values lets us define successive terms recursively:
             d0 = d1
             a0 = a1
             b0 = b1
+            count--
         }
-        return Frac(sign * n0, d0, err)
+        return if (count > 0) Frac(sign * n0, d0, err) else Frac(0,0, 0.0)
```

Comment:
might want to document what failure returns and why we have to handle it (because of large or small enough input won't resolve to 0 error)